sapp/ui/filter_predicates.py

```diff
@@ -5,8 +5,19 @@
 
 from __future__ import annotations
 
+import js_regex
```

Comment:
nitpick: since this is not in the standard library, I would put it in the next import block (next to sqlalchemy stuff)

Reply:
Yep moving it :)